lib/services/beanstalk/index.js

```diff
@@ -295,7 +295,7 @@ exports.deploy = function (ownServiceContext, ownPreDeployContext, dependenciesD
         .then(serviceRole => {
             return getSystemInjectedEbExtensions(stackName, ownServiceContext, dependenciesDeployContexts)
                 .then(ebextensionFiles => {
-                    return deployableArtifact.prepareAndUploadDeployableArtifact(ownServiceContext, ebextensionFiles)
+                    return deployableArtifact.prepareAndUploadDeployableArtifact(accountConfig, ownServiceContext, ebextensionFiles)
```

Comment:
Likewise in this file, you don't need to pass accountConfig into the method, just require it inside that file.